strings/anagrams.py

```diff
@@ -18,23 +18,23 @@ def signature(word: str) -> str:
 
 
 def anagram(my_word: str) -> list[str]:
-    """Return every anagram of the given word
+    """Returns every anagram of the given word
     >>> anagram('test')
     ['sett', 'stet', 'test']
     >>> anagram('this is a test')
     []
     >>> anagram('final')
     ['final']
     """
-    return word_bysig[signature(my_word)]
+    return word_by_sig[signature(my_word)]
```

Comment:
```suggestion
    return word_by_signature[signature(my_word)]
```
What is your opinion on renaming `word_by_sig` to `word_by_signature`?